applications/MeshingApplication/tests/mmg_lagrangian_test/beam2D_test_result.mesh

```diff
@@ -0,0 +1,952 @@
+MeshVersionFormatted 2
```

Comment:
It is a result used to check that the remeshing is correct, any other suggestion?